crates/net/eth-wire/src/capability.rs

```diff
@@ -115,7 +122,7 @@ impl Capabilities {
     /// Whether the peer supports `eth` sub-protocol.
     #[inline]
     pub fn supports_eth(&self) -> bool {
-        self.eth_67 || self.eth_66
+        self.eth_67 || self.eth_66 || self.eth_68
```

Comment:
since we expect that client will update over time we can change the order here: 68 || 67 || 66

Reply:
will do

Reply:
follow it up with b7b4b147cb2d222575d16c35d6dcfb36d16b335b. nit but please note that I also changed the eth versions order as desc in `HelloMessageBuilder`.